components/support/nimbus-cli/src/main.rs

```diff
@@ -471,6 +487,8 @@ pub(crate) struct AppOpenArgs {
     passthrough: Vec<String>,
     pbcopy: bool,
     pbpaste: bool,
+
+    output: Option<PathBuf>,
```

Comment:
Nit: I think this is an extra line
```suggestion
    pbpaste: bool,
    output: Option<PathBuf>,
```